### PR TITLE
Pass screen width in a variable to relay resize artworks query

### PR DESCRIPTION
**Pull request by ashleyjelks**
Follow up to #2032.

Instead of resizing the artworks images based on a static width of 500 px, this PR adds a variable to the Featured Artists Collections relay query to resize images based on the screen width.

### CODE REVIEW

File: src/lib/relay/QueryRenderers.tsx [hunk 1]
```diff
@@ -280,24 +281,28 @@ interface CollectionRendererProps extends RendererProps {
 }
 
 export const CollectionRenderer: React.SFC<CollectionRendererProps> = ({ collectionID, render }) => (
-  <QueryRenderer<QueryRenderersCollectionQuery>
-    environment={environment}
-    query={graphql`
-      query QueryRenderersCollectionQuery($collectionID: String!) {
-        collection: marketingCollection(slug: $collectionID) {
-          ...Collection_collection
+  console.log(Dimensions.get("screen").width, "SCREEEEEEEEN WIDTH"),
```
Comment (pepopowitz) [inline src/lib/relay/QueryRenderers.tsx]: Rogue `console.log`. (I'd use the suggestion feature, but it is multiple lines and I don't think the suggestion feature supports that)

Comment (ashleyjelks) [reply src/lib/relay/QueryRenderers.tsx]: Good catch!

**Comment by ashleyjelks** [comment]
> Very cool! This is just what I had in mind.
> 
> One thing that would make this really great is a unit test. [Here](https://github.com/artsy/emission/blob/22183b75cf09c9692bd6fa80d6b55b827a3e010c/src/lib/Scenes/Artwork/__tests__/hardware-tests.ts#L1-L3) is an example of how you can mock things in the `react-native` module (like `Dimensions` in this case) and [here](https://github.com/artsy/emission/blob/22183b75cf09c9692bd6fa80d6b55b827a3e010c/src/lib/Scenes/Search/__tests__/AutosuggestResults-tests.tsx#L162) is an example of how to test that a query variable is passed through. I wouldn't say that the unit test is _necessary_ before merging, since the code isn't _too_ complex, but the latter is an example of using [Relay's new Test Utils](https://relay.dev/docs/en/testing-relay-components), which we're trying to standardize on, so it could be useful as practice.

I think this is a great idea! I'll make a ticket to track the work.

**Comment by artsyit** [comment]
:rocket: PR was released in v1.21.7 :rocket:

**Pull request merged**